src/components/PlanetDetails/PlanetDetails.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  useLoaderData,
+  useNavigate,
+  useNavigation,
+  useParams,
+} from 'react-router-dom';
+import { Planet } from '../../services/searchService/types';
+import Spinner from '../Spinner/Spinner';
+
+import styles from './planetDetails.module.css';
+
+export default function PlanetDetails() {
+  const { planetName } = useParams();
+  const planetDetails = useLoaderData() as Planet;
+  const navigation = useNavigation();
+
+  const navigate = useNavigate();
+
+  const {
+    diameter,
+    gravity,
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    orbital_period,
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    rotation_period,
+    climate,
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    surface_water,
+    terrain,
+    population,
+  } = planetDetails;
+
+  return (
+    <div className={styles.planetDetail__wrapper}>
+      {navigation.state === 'loading' && <Spinner />}
+      {navigation.state !== 'loading' && (
+        <>
+          {' '}
```

Comment:
Is it posible to define these gaps with css?

Reply:
fixed, changed to margins